docs/site/content/docs/latest/tanzu-diagnostics.md

```diff
@@ -113,3 +119,22 @@ This command will attempt to collect diagnostics data for any boostrap cluster t
 ```sh
 tanzu diagnostics collect --bootstrap-cluster-skip --management-cluster-skip --workload-cluster-name=<WORKLOAD_CLUSTER_NAME>
 ```
+
+## Collecting unmanaged cluster diagnostics
+
+Finally, the plugin also supports collecting diagnostics from unmanaged-clusters, a lite weight, local deployment model.
+Because unmanaged-clusters do _not_ have a bootstrap cluster and are _not_ managed by a tanzu management cluster,
+you must provide the name of the unmanaged cluster and the context:
+
+```sh
+tanzu diagnostics collect --unmanaged-cluster-name kind-my-unmanaged-cluster --unmanaged-cluster-context kind-my-unmanaged-cluster
+```
+
+_Note:_ The default provider for unmanaged-clusters is Kind. This means that by default, the name of the unmanaged-cluster will
+have `kind-` prefixed.
+
+If you need to see what the name of your cluster is and it's context, while targeting your unmanged cluster, run:
+
+```sh
+kubectl config view --minify
```

Comment:
I wonder if it would be better to document using `kubectl config get-contexts`. The minify command gives an error if your currently set config doesn't exist, which is currently the case on my machine:

```
❯ k config view --minify
error: current-context must exist in order to minify
```

Whereas `get-contexts` does give a fairly concise table output:

```
❯ k config get-contexts
CURRENT   NAME                    CLUSTER     AUTHINFO        NAMESPACE
          gh2159-admin@gh2159     gh2159      gh2159-admin
          kind-kind               kind-kind   kind-kind
          tfmgmt-admin@tfmgmt     tfmgmt      tfmgmt-admin
          v10test-admin@v10test   v10test     v10test-admin
```

Reply:
This was resolved but didn't change. Did you intend to?

Reply:
I added these lines _afterwards_. Thoughts? Does that work? https://github.com/vmware-tanzu/community-edition/pull/3469/files#diff-09909f3496f63fd28a87dc7f2f14b7d1fdb6dae29a2a3fc02e99162498d4d05bR140-R152

Reply:
I guess so, but with needing a special note and instructions for someone to make sure their context is set first, plus the format of the output, I think I still would prefer to just use `kubectl get-contexts` and keep it simple.

Looks like my contexts are stale too, because even following the recommendation, I get errors trying to run the command.

Let's see if we can get any other feedback.

Reply:
no input on my end (ie don't block on me)

i'm fine with whatever ya'll come up with.

Reply:
Fair enough. Let's see if anyone complains that it doesn't work for them, then change it if it's actually an issue.